core/services/log/broadcaster.go

```diff
@@ -223,19 +237,19 @@ func (b *broadcaster) startResubscribeLoop() {
 		}
 
 		var backfillFrom null.Int64
-		if b.latestHeadFromDb != nil {
+		if b.requestedBackfillFrom.Valid {
 			// The backfill needs to start at an earlier block than the one last saved in DB, to account for:
```

Comment:
think this comment needs to be updated?

Reply:
actually i left the comment in, but still kept the explicit head from db. It's now a bit simpler i think, and replay is more precise